async-nats/src/jetstream/object_store/mod.rs

```diff
@@ -259,11 +259,11 @@ impl ObjectStore {
         let mut object_chunks = 0;
         let mut object_size = 0;
 
-        let mut buffer = [0; DEFAULT_CHUNK_SIZE];
+        let mut buffer = Box::new([0; DEFAULT_CHUNK_SIZE]);
```

Comment:
Just do a vec?

Reply:
few bytes saved on not having `len` field? 😄